database/influxdb/influxdb_database.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+
+# (c) 2016, Kamil Szczygiel <kamil.szczygiel () intel.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: influxdb_database
+short_description: Manage InfluxDB databases
+description:
+    - Manage InfluxDB databases
+version_added: 2.1
```

Comment:
@kamsz: The door has been closed on v2.1 that you'll need to update this to at least 2.2 for Travis to start passing again. We may need to pull the retention policy module out since it doesn't make much sense to ship that module without this one.